.github/workflows/ci.yml

```diff
@@ -81,3 +81,21 @@ jobs:
         run: cargo install cargo-apk
       - name: Build APK
         run: cargo apk build --example android
+
+  markdownlint:
```

Comment:
I suggest adding the markdown lint as a conditional step in the CI job, similar to `cargo fmt` on lines 37-39.

Pros:
- Helps avoid hitting the global max 10 simultaneous jobs limit per free GitHub organization
- Avoids a relatively lengthy setup and clone to run a (presumably) very fast markdown check

Cons:
- Requires adding an `if: runner.os == 'linux' && matrix.toolchain == 'stable'` line to the `Run Markdown Lint` step
- Makes the CI job on linux (already our longest job by wall clock time) even longer.

If my assumption is correct that markdown linting takes only seconds, I think this change would be worth the reduced parallelism to avoid the overhead of a separate job.

Reply:
I made it a separate Job, as I saw the Build Jobs solely responsible for the Rust testing.
It would be annoying to get no Compile Status for different OS Types because an (unimportant) Markdown File was wrong.

That's why I made it a seperate Job.

Reply:
I reviewed the actual timing of the CI runs of the current code.

The markdown linting job duration is almost entirely dominated by retrieving the super linter container (~2m) and _everything else_ including the linting itself takes a tiny fraction of time (~20s).

I don't think the usage of an extra job slot outweighs the ~2.5m runtime increase this would add to the linux build. So, I retract my suggestion. "Measure things" FTW. Also, it's clearer as you mentioned. 😄